File: src/main/java/de/kevcodez/metronom/MetronomApp.java
package de.kevcodez.metronom;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class MetronomApp {

    public static void main(String[] args) {
        SpringApplication.run(MetronomApp.class, args);
    }
}

File: src/main/kotlin/de/kevcodez/metronom/MetronomApp.kt
package de.kevcodez.metronom

import org.springframework.boot.Banner
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
open class MetronomApp

fun main(args: Array<String>) {
    runApplication<MetronomApp>(*args) {
        setBannerMode(Banner.Mode.OFF)
    }
}